com.unity.render-pipelines.universal/Documentation~/TableOfContents.md

```diff
@@ -18,6 +18,7 @@
     * [Decal](renderer-feature-decal.md)
       * [Decal Shader Graph](decal-shader.md)
 * [Upgrade guides](upgrade-guides.md)
+  * [Render Pipeline Converter](features/rp-converter.md)
   * [Upgrading to URP 12.0.x](upgrade-guide-2021-2.md)
```

Comment:
Imo there should also be an entry in the upgrading to 12.0.x that mentions the old menu is gone and users shoudl use the new rp converter.

Reply:
Thanks, I've added the point (and also a reference at the top of the page).